docs_user/adoption-attributes.adoc

```diff
@@ -5,10 +5,14 @@
 ifeval::["{build}" == "upstream"]
 :OpenShift: OpenShift
 :osp_prev_long: OpenStack
+:osp_acro: OSP
 :OpenStackShort: OSP
 :OpenShiftShort: OCP
 :osp_curr_ver: Antelope
+:osp_prev_ver: Wallaby
 :OpenStackInstaller: TripleO
+:CephService_first_ref: Manila
```

Comment:
for the upstream version, using two variables w/ the same name can be optimize w/ a single variable: basically `rhos_component_storage_file` and `rhos_component_storage_file_first_ref` can be converged to `rhos_component_storage_file`.
@klgill not sure if I'm messing up w/ the pattern you're trying to establish but it was just out of curiosity, I'm ok to keep them separated if it makes the flow cleaner.

Reply:
I would love to consolidate these, but the issue is with the first reference of the service in the downstream docs. For example, I could set the value for the "rhos_component_storage_file" to "Manila" for upstream and "Shared File System service (manila)" for downstream, but we only need "(manila)" on the first reference in the downstream documentation. 

On the other hand, if I use the variable and manually type "(manila)" after it, the docs will render as "Manila (manila)" upstream.

I'll ask my team for advice on this.